db/fixtures/ae_datastore/ManageIQ/ConfigurationManagement/AnsibleTower/Service/Provisioning/StateMachines/Provision.class/__methods__/preprovision.rb

```diff
@@ -0,0 +1,26 @@
+#
+# Description: This method prepares arguments and parameters for a job template
+#
+
+$evm.log("info", "Starting Ansible Tower Pre-Provisioning")
+
+service = $evm.root["service_template_provision_task"].destination
+
+# Through service you can examine the job template, configuration manager (i.e., provider)
+# and options to start a job
+# You can also override these selections through service
+
+# $evm.log("info", "manager = #{service.configuration_manager.name}(#{service.configuration_manager.id})")
+# $evm.log("info", "template = #{service.job_template.name}(#{service.job_template.id}))")
+
+# Caution: stack_options may contain passwords.
+# $evm.log("info", "stack options = #{service.stack_options.inspect}")
+
+# Example how to programmatically modify stack options:
+# service.stack_name = 'new_name'
+# stack_options = service.stack_options
+# stack_options[:disable_rollback] = false
+# stack_options[:timeout_mins] = 2 # this option is provider dependent
+# stack_options[:parameters]['flavor'] = 'm1.small'
```

Comment:
Yes. This is to be corrected in the next PR, when we support dialog for options.